mmv1/products/orgpolicy/Policy.yaml

```diff
@@ -0,0 +1,204 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: Policy
+base_url: '{{parent}}/policies'
+create_url: '{{parent}}/policies'
+self_link: '{{parent}}/policies/{{name}}'
+import_format:
+  ['{{%parent}}/policies/{{name}}']
```

Comment:
We don't have long form handling here, is that because parent is necessarily a long form?

Reply:
Yes, parent always needs to have `organizations/`, `projects/` or `folders/`